spdx/parsers/rdf.py

```diff
@@ -396,6 +398,13 @@ def p_pkg_comment(self, p_term, predicate):
         except CardinalityError:
             self.more_than_one_error("package comment")
 
+    def p_pkg_has_file(self, p_term, predicate):
+        for _, _, has_file in self.graph.triples((p_term, predicate, None)):
+            for index, spdx_file in enumerate(self.doc.unpackaged_files):
+                if str(has_file) == spdx_file.spdx_id:
+                    self.doc.packages[-1].files.append(spdx_file)
+                    del self.doc.unpackaged_files[index]
```

Comment:
Not sure if it fits with the rest of the style of the code, but a comment here would definitely be helpful. It's hard to interpret what `has_file` actually is, and reads kind of weird when it is compared directly against `spdx_file.spdx_id`